src/components/SubmittedForms/FormStatus/FormStatus.tsx

```diff
@@ -21,16 +29,27 @@ const FormStatus = ({ id, applicationStatus }: FormStatus) => {
           options={FORM_STATUS_DROPDOWN_OPTIONS}
           selectedValue={
             applicationStatus === 'PENDING'
-              ? 'SELECT'
+              ? 'PENDING'
               : applicationStatus === 'CONFIRMED'
-              ? 'CONFIRM'
-              : 'SELECT'
+              ? 'CONFIRMED'
+              : 'PENDING'
           }
-          // onChange={} : #TODO :선택값 바뀌면 mutation 호출
+          onChange={(event) => {
+            changeSubmittedFormStatus({
+              eventId,
+              formUserId: id,
+              status: event.target.value as EventStatus,
+            });
+          }}
+          disabled={applicationStatus === 'CANCELED'}
         />
       </FormStatusItemStyled>
       <FormStatusItemStyled>
-        <ApplyCancelButton isCanceled={applicationStatus === 'CANCELED' ? true : false} />
+        <ApplyCancelButton
+          isCanceled={applicationStatus === 'CANCELED' ? true : false}
```

Comment:
p4;
```suggestion
          isCanceled={applicationStatus === 'CANCELED'}
```
위에 disabled에 준것과 같이 어느쪽이든 통일하면 좋겠네요!